### PR TITLE
Add classes person student teacher

### PR DESCRIPTION
**Pull request by tiagomarin**
In this pull request I added:

- Created class Person, with attributes age, name, and parent_permission.
- Added methods getters and setters, can_use_services, and of_age to class Person.
- Created class Student inherited from class Person and extended the method initialize to include attribute classroom.
- Created class Teacher inherited from class Person and extended the method initialize to include attribute specialization.
- Added methods play_hooky to class Student, and override method can_use_services in class Teacher.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student
```
Comment (OLIPLICHE) [inline student.rb]: - [ ] The requirement says `Student` Inherits from `Person`. You need a code like this: `class Student < Person`
- [ ] Please [open this link to see examples](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance.)

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher
```
Comment (OLIPLICHE) [inline teacher.rb]: - [ ] `Inherits from Person`. Kindly apply the same Hint and comment I wrote in the student class. Also, you can refer to the project requirements.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @tiagomarin, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Implement the classes to represent students and teachers✅✅
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (tiagomarin) [reply student.rb]: Thanks a lot, I totally forgot to inherit the super class!

File: student.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+require './person'
```
Comment (rloterh) [inline student.rb:1]: - [x] OPTIONAL
By convention, it's a good practice to use `require_relative` classes in the same source and use `require` for external modules or packages.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,53 @@
...
+- Run the app.rb file on the terminal and follow the step by step answering the questions.
+
+
+### Prerequisites
+
+- Ruby installed in your computer. ([check this guide if you need to install](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md))
+- Any code editor like Vscode or Vim.
+
+
+## Authors
```
Comment (rloterh) [inline README.md:29]: - [x] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,53 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Part of Microverse curriculum
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rloterh) [inline README.md:53]: - [x] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it.

File: student.rb (hunk 1 above)
Comment (rloterh) [reply student.rb]: well resolved ✔️

File: teacher.rb (hunk 2 above)
Comment (rloterh) [reply teacher.rb]: well resolved ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Tiago,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Followed best OOP practice

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 3 above)
Comment (tiagomarin) [reply student.rb:1]: Thanks a lot I'll do that!

File: README.md (hunk 4 above)
Comment (tiagomarin) [reply README.md:29]: Thanks for pointing that out! I'll apply this change also!

File: README.md (hunk 5 above)
Comment (tiagomarin) [reply README.md:53]: Again, thanks for pointing it out!

**Pull request merged**